sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ReadmeSamples.java

```diff
@@ -341,7 +348,153 @@ public void databaseCreateContainerSample() {
             System.out.println("Failed to create container: " + ce);
         }
         // END: com.azure.cosmos.CosmosDatabase.createContainer
+    }
+
+
+    public void getDatabaseSample() {
+        CosmosAsyncClient cosmosAsyncClient = new CosmosClientBuilder()
+            .buildAsyncClient();
+        // BEGIN: com.azure.cosmos.CosmosAsyncClient.getDatabase
+        String databaseId = "<YOUR DATABASE NAME>";
+
+        CosmosAsyncDatabase database = cosmosAsyncClient.getDatabase(databaseId);
+        // END: com.azure.cosmos.CosmosAsyncClient.getDatabase
+    }
+
+    public void createGlobalThroughputControlConfigBuilderSample() {
+        CosmosAsyncClient cosmosAsyncClient = new CosmosClientBuilder()
+            .buildAsyncClient();
+        // BEGIN: com.azure.cosmos.CosmosAsyncClient.createGlobalThroughputControlConfigBuilder
+        String databaseId = "<YOUR DATABASE NAME>";
+        String containerId = "<YOUR CONTAINER NAME>";
+
+        GlobalThroughputControlConfig config =
+            cosmosAsyncClient.createGlobalThroughputControlConfigBuilder(databaseId, containerId)
+                .build();
+        // END: com.azure.cosmos.CosmosAsyncClient.createGlobalThroughputControlConfigBuilder
+    }
+
+    public void readConflictSample() {
+        // BEGIN: com.azure.cosmos.CosmosAsyncConflict.read
+        String conflictId = "CONFLICT_ID";
+        CosmosAsyncDatabase cosmosAsyncDatabase =
+            new CosmosAsyncDatabase(conflictId, cosmosAsyncClient);
```

Comment:
This is not the right approach to create the object, we should rather get it from cosmosAsyncClient.